src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -486,15 +488,39 @@ protected function backfill_multiday_event_ids( array $events = [] ) {
 				continue;
 			}
 
+			$site_timezone     = Timezones::build_timezone_object();
+			$utc               = Timezones::build_timezone_object( 'UTC' );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 1294710ee261d20026b65e2d478066a7</sub>